dgraph/cmd/zero/http.go

```diff
@@ -231,9 +234,46 @@ func (st *state) getState(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (s *Server) zeroHealth(ctx context.Context) (*api.Response, error) {
+	if ctx.Err() != nil {
+		return nil, ctx.Err()
+	}
+	health := pb.HealthInfo{
+		Instance: "zero",
+		Address:  x.WorkerConfig.MyAddr,
+		Status:   "healthy",
+		Version:  x.Version(),
+		Uptime:   int64(time.Since(x.WorkerConfig.StartTime) / time.Second),
+		LastEcho: time.Now().Unix(),
+	}
+	jsonOut, err := json.Marshal(health)
+	if err != nil {
+		return nil, errors.Errorf("unable to marshal. error %+v", err)
```

Comment:
can you add more detail to the error, something like unable to marshal zero health. It makes finding the error in code much easier.